docs/self-hosting/guides/automated-bootstrapping.mdx

```diff
@@ -0,0 +1,160 @@
+```
+
+This command pipes the output through `jq` to extract only the machine identity token, making it easy to capture and use directly in automation scripts or export as an environment variable for tools like Terraform.
+
+## API Response Structure
+
+The bootstrap process returns a JSON response with details about the created user, organization, and machine identity:
+
+```json
+{
+  "identity": {
+    "credentials": {
+      "token": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZGVudGl0eUlkIjoiZGIyMjQ3OTItZWQxOC00Mjc3LTlkYWUtNTdlNzUyMzE1ODU0IiwiaWRlbnRpdHlBY2Nlc3NUb2tlbklkIjoiZmVkZmZmMGEtYmU3Yy00NjViLWEwZWEtZjM5OTNjMTg4OGRlIiwiYXV0aFRva2VuVHlwZSI6ImlkZW50aXR5QWNjZXNzVG9rZW4iLCJpYXQiOjE3NDIzMjI0ODl9.mqcZZqIFqER1e9ubrQXp8FbzGYi8nqqZwfMvz09g-8Y"
+    },
+    "id": "db224792-ed18-4277-9dae-57e752315854",
+    "name": "Instance Admin Identity"
+  },
+  "message": "Successfully bootstrapped instance",
+  "organization": {
+    "id": "b56bece0-42f5-4262-b25e-be7bf5f84957",
+    "name": "dog",
+    "slug": "dog-v-e5l"
+  },
+  "user": {
+    "email": "admin@example.com",
+    "firstName": "Admin",
+    "id": "a418f355-c8da-453c-bbc8-6c07208eeb3c",
+    "lastName": "User",
+    "superAdmin": true,
+    "username": "admin@example.com"
+  }
+}
+```
+
+## Using the Instance Admin Machine Identity Token
+
+The bootstrap process automatically creates a machine identity with Token Auth configured. The returned token has instance-level admin privileges (the highest level of access) and should be treated with the same security considerations as a root credential.
+
+The token enables full programmatic control of your Infisical instance and can be used in the following ways:
+
+### 1. Infrastructure Automation
+
+Store the token securely for use with infrastructure automation tools. Due to the sensitive nature of this token, ensure it's protected using appropriate secret management practices:
+
+#### Kubernetes Secret (with appropriate RBAC restrictions)
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: infisical-admin-credentials
+type: Opaque
+data:
+  token: <base64-encoded-token>
+```
+
+#### Environment Variable for Terraform
+
+```bash
+export INFISICAL_TOKEN=your-access-token
+terraform apply
+```
+
+### 2. Programmatic Resource Management
+
+Use the token to authenticate API calls for creating and managing Infisical resources. The token works exactly like any other Token Auth access token in the Infisical API:
+
+```bash
+curl -X POST \
+  -H "Authorization: Bearer ${INFISICAL_TOKEN}" \
+  -H "Content-Type: application/json" \
+  -d '{
+    "projectName": "New Project",
+    "projectDescription": "A project created via API",
+    "slug": "new-project-slug",
+    "template": "default",
+    "type": "SECRET_MANAGER"
+  }' \
+  https://your-infisical-instance.com/api/v2/projects
+```
+
+## End-to-End Workflow
```

Comment:
i think we can review this. feels like i already read this earlier and might be redundant?